src/components/dashboard/main-content/course-enrollments/course-cards/unenroll/UnenrollModal.jsx

```diff
@@ -33,6 +40,33 @@ const UnenrollModal = ({
     onClose();
   };
 
+  const updateQueryForUnenrollment = () => {
+    const enrollmentForCourseFilter = (enrollment) => enrollment.courseRunId !== courseRunId;
+    const isBFFEnabled = isBFFEnabledForEnterpriseCustomer(enterpriseCustomer.uuid);
+    // Determine which BFF queries need to be updated after unenrolling.
```

Comment:
nit: i think this comment relates mostly to stuff within the `isBFFEnabled` conditional. maybe move it within that block?